gym_xiangqi/constants.py

```diff
@@ -3,11 +3,14 @@
 the Xiangqi environment.
 """
 
+import os
+
 """ PATHS """
-PATH_TO_SOUNDS = "gym_xiangqi/sounds/"
-PATH_TO_BOARD = "gym_xiangqi/images/board/"
-PATH_TO_BLACK = "gym_xiangqi/images/black_pieces/"
-PATH_TO_RED = "gym_xiangqi/images/red_pieces/"
+ROOT_DIR = os.path.dirname(__file__)
+PATH_TO_SOUNDS = os.path.join(ROOT_DIR, "sounds/")
+PATH_TO_BOARD = os.path.join(ROOT_DIR, "images/board/")
+PATH_TO_BLACK = os.path.join(ROOT_DIR, "images/black_pieces/")
+PATH_TO_RED = os.path.join(ROOT_DIR, "images/red_pieces/")
```

Comment:
Will this change be necessary? I think it will be better if we can leave them as string literals instead of using the `os` library calls. I also think relative imports will be fine since everything will be packaged within our project root directory.